client/modules/IDE/components/FileNode.jsx

```diff
@@ -156,6 +168,8 @@ export class FileNode extends React.Component {
                   ref={(element) => { this[`fileOptions-${this.props.id}`] = element; }}
                   tabIndex="0"
                   onClick={this.toggleFileOptions}
+                  onBlur={this.blurComponent}
+                  onFocus={() => { this.setState({ isFocused: true }); }}
```

Comment:
could you move this into a function called `onfocusComponent`?

Reply:
Definitely! I will fix this now.